helm-chart/values.yaml

```diff
@@ -1,25 +1,43 @@
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
+
 adminPassword: admin
 image:
   repository: "molgenis/molgenis-emx2"
   tag: "latest"
   pullPolicy: IfNotPresent
 
+database:
+  name: molgenisdb
```

Comment:
I mean, this code would not be nessary if we wouldn't have added this flexibility? Why do we put this burden on us? Was there a use case that triggers this? I prefer convention over configuration